### ISSUE TITLE
Unable to run KLI command line - Unable to find libsodium

### ISSUE THREAD

**Issue by andrewBSTan** [WebOfTrust/keripy#487]
I have libsodium installed on my Mac and has added to the path, but I am getting an error of "Unable to find libsodium" when I run kli version.

$PATH
/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/andrewtan//opt/homebrew/Cellar/libsodium/1.0.18_1

Any advice on where I could have gone wrong?

Andy

**Comment by 2byrds** [comment]
Hi @andrewBSTan , I recently went through @kentbull excellent guide on my Mac and it worked the first time. I recommend it! https://kentbull.com/2023/03/09/keri-tutorial-series-treasure-hunting-in-abydos-issuing-and-verifying-a-credential-acdc/

**Comment by kentbull** [comment]
@andrewBSTan if you did `brew install libsodium` then that is likely your problem. That never worked for me which is why I instructed my blog readers, as @2byrds mentioned, to install directly from a source tarball.

Here are the instructions repeated for your convenience:

- Libsodium
  - https://libsodium.gitbook.io/doc/installation
  - On my MacBook Pro I ended up having to do the full installation:
  - Download a tarball from the [releases page](https://download.libsodium.org/libsodium/releases/).
  - Change directories to within the decompressed tarball.
```bash
$ ./configure
$ make && make check
$ sudo make install
```
